mk/docs.mk

```diff
@@ -37,12 +37,31 @@ docs/generated/openapi.yaml:
 	mkdir -p $(dir $@)
 	mkdir -p $(OAPI_TMP_DIR)/policies
 	mkdir -p $(OAPI_TMP_DIR)/resources
+
+	for resource in Mesh MeshGateway; do \
+  		lowercase=$$(echo $${resource} | tr '[:upper:]' '[:lower:]'); \
+		mkdir -p $(OAPI_TMP_DIR)/resources/$${lowercase}/; \
+		go run tools/template/main.go --template tools/policy-gen/templates/schema.yaml --data '{"Scope": "Global"}' | \
+		$(YQ) -o json | \
+		$(JQ) --slurpfile defs ./docs/generated/raw/protos/$${resource}.json -f tools/template/resolve_refs.jq | \
+		$(YQ) -P > $(OAPI_TMP_DIR)/resources/$${lowercase}/schema.yaml ; \
+		go run tools/template/main.go --template tools/policy-gen/templates/endpoints.yaml --data "{\"Package\": \"v1.0.0\", \"Name\": \"$${resource}\", \"Scope\": \"Global\", \"Path\": \"$${lowercase}es\"}" > $(OAPI_TMP_DIR)/resources/$${lowercase}/rest.yaml; \
```

Comment:
One of the reason of the generator was the ability to have access to the [ResourceInfo](https://github.com/kumahq/kuma/blob/2ad2aebba8d59df93a6dd42e55dbae1095efdd3d/tools/resource-gen/genutils/util.go#L45-L69)  then you could use `WsPath` for .e.g